jpsonic-main/src/main/java/com/tesshu/jpsonic/dao/MediaFileDao.java

```diff
@@ -1158,4 +1191,7 @@ Optional<String> getShowUnstarredSongsCondition() {
             return Optional.empty();
         }
     }
+
+    public record IndexWithCount(String index, int artistCount) {
+    }
```

Comment:
count or directoryCount is better instead of artistCount. Album is also counted.

Reply:
Done.